packages/api/tests/lambdas/test-sqs-message-consumer.js

```diff
@@ -362,3 +364,85 @@ test.serial('SQS message consumer queues correct number of workflows for rules m
     await cleanupRulesAndQueues(rules, [queue]);
   });
 });
+
+test.serial('processQueues archives messages from the ENABLED sqs rule only', async (t) => {
+  const { stackName } = process.env;
+  const { rules, queues } = await createRulesAndQueues();
+  const message = { testdata: randomString() };
+
+  // Send message to ENABLED queue
+  const firstMessage = await SQS.sendSQSMessage(
+    queues[0].queueUrl,
+    message
+  );
+
+  // Send message to DISABLED queue
+  const secondMessage = await SQS.sendSQSMessage(
+    queues[1].queueUrl,
+    { testdata: randomString() }
+  );
+  const enabledQueueKey = getS3KeyForArchivedMessage(stackName, firstMessage.MessageId);
+  const deadLetterKey = getS3KeyForArchivedMessage(stackName, secondMessage.MessageId);
+
+  await handler(event);
+
+  const item = await s3().getObject({
+    Bucket: process.env.system_bucket,
+    Key: enabledQueueKey,
+  }).promise();
+
+  t.deepEqual(message, JSON.parse(JSON.parse(item.Body.toString())));
+
+  await t.throwsAsync(s3().getObject({
```

Comment:
Use `t.false` with `s3ObjectExists`. The reason I'm asking for this is that it's more clear what the test is checking for and we don't need to compare against a specific error coe